Pod/Classes/PagingMenuOptions.swift

```diff
@@ -20,6 +20,7 @@ public class PagingMenuOptions {
     public var menuPosition: MenuPosition = .Top
     public var menuHeight: CGFloat = 50
     public var menuItemMargin: CGFloat = 20
+    public var menuItemDividerImage: UIImage! = nil
```

Comment:
I have changed to ?. I'm fairly new to Swift so I sometimes still mix up when to use.